crates/bevy_pbr/src/meshlet/meshlet_cull_shared.wgsl

```diff
@@ -0,0 +1,207 @@
+#define_import_path bevy_pbr::meshlet_cull_shared
+
+#import bevy_pbr::meshlet_bindings::{
+    MeshletAabb,
+    DispatchIndirectArgs,
+    InstancedOffset,
+    depth_pyramid,
+    view,
+    previous_view,
+    meshlet_instance_uniforms,
+}
+#import bevy_render::maths::affine3_to_square
+
+// https://github.com/zeux/meshoptimizer/blob/1e48e96c7e8059321de492865165e9ef071bffba/demo/nanite.cpp#L115
+fn lod_error_is_imperceptible(lod_sphere: vec4<f32>, simplification_error: f32, instance_id: u32) -> bool {
+    let world_from_local = affine3_to_square(meshlet_instance_uniforms[instance_id].world_from_local);
+    let world_scale = max(length(world_from_local[0]), max(length(world_from_local[1]), length(world_from_local[2])));
+    let camera_pos = view.world_position;
+
+    let projection = view.clip_from_view;
+    if projection[3][3] == 1.0 {
+        // let world_sphere_radius = lod_sphere.w * world_scale;
+        // let norm_error = simplification_error / world_sphere_radius * 0.25;
+        // return norm_error * view.viewport.w < 1.0;
+        let world_error = simplification_error * world_scale;
+        let proj = projection[1][1];
+        let height = 2.0 / proj;
+        let norm_error = world_error / height;
+        return norm_error * view.viewport.w < 1.0;
```

Comment:
Unsure how correct my change (from the commented code) is, though it seems to work.